pynestml/src/main/grammars/org/Tokens.g4

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Tokens.g4
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+lexer grammar Tokens;
+
+  SL_COMMENT : ('#' (~('\n' |'\r' ))*|
```

Comment:
What is about multi-line comments?

Reply:
The second alternative ensures that comments can be multiline:
```  
SL_COMMENT : ('#' (~('\n' |'\r' ))*| 
               '/*' .*? '*/' | '"""' .*? '"""')
               -> channel(HIDDEN)
             ;
```
Here, we allow Java style comments /* ... */ and Python style comments """...""" .

Reply:
However, currently, comments are not further regarded. This is a part of future work which will deal with those parts.

Reply:
Hi, but it would be better, that you either have 2 productions `SL_COMMENTS` and `ML_COMMENTS` or only one which is `COMMENTS`.

Reply:
Assuming SL stands for Single-Line, then you are right =).